### PR TITLE
Kingmushroom patch 1

### PR DESCRIPTION
**Pull request by kingmushroom**
fixes the (ideally) last few bugs.

### CODE REVIEW

File: serverless.yml [hunk 1]
```diff
@@ -43,7 +43,7 @@ functions:
       checkpoint: 3
       bucket_name: results-s3-bucket
       identifier_column: responder_id
-      in_file_name: ingest_out.json
+      in_file_name: test_data_mike_1.json
```
Comment (KrisRogos) [inline serverless.yml:46]: Ummm... is this meant to be test_data_mike_1 instead of ingest_out?

**Review by KrisRogos — CHANGES_REQUESTED** [review]
Can you confirm if this change was desired or by accident?

File: serverless.yml (hunk 1 above)
Comment (kingmushroom) [reply serverless.yml:46]: yes, data from ingest is too small. Ian n Paul are aware and happy with this till our input data is better

Comment (KrisRogos) [reply serverless.yml:46]: Cool, I'll approve then.

**Review by KrisRogos — APPROVED** [review]

**Review by thomashenson — APPROVED** [review]
OK to merge.

**Pull request merged**